atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -1077,7 +1078,7 @@ private static LockAndTimestampServices getLockAndTimestampServices(
         NamespacedConjureTimelockService namespacedConjureTimelockService
                 = new NamespacedConjureTimelockService(withDiagnosticsConjureTimelockService, timelockNamespace);
 
-        LockWatchEventCache lockWatchEventCache = NoOpLockWatchEventCache.INSTANCE;
+        LockWatchEventCache lockWatchEventCache = LockWatchEventCacheImpl.create();
```

Comment:
Sanity check: This _should_ be the only place where we need to wire in the real cache over the NoOp one (I checked this by looking at where `LockWatchManagerImpl` is created, and it appears the only real one is here).

Reply:
Yep.